packages/core/src/login/webview/vue/login.vue

```diff
@@ -581,44 +593,59 @@ export default defineComponent({
 }
 .urlInput {
     background-color: var(--vscode-input-background);
-    width: 100%;
-    height: 15px;
-    color: white;
+    width: 244px;
+    height: 28px;
+    box-sizing: border-box;
     border: none;
-}
-.vscode-light .urlInput {
+    padding-left: 8px;
+    padding-right: 8px;
+    padding-top: 6px;
+    padding-bottom: 6px;
+    font-size: 13px;
+    font-weight: 400;
+}
+body.vscode-light .urlInput {
     color: black;
 }
-.vscode-dark .urlInput {
-    color: white;
+body.vscode-dark .urlInput {
+    color: #cccccc;
 }
 .iamInput {
     background-color: var(--vscode-input-background);
-    width: 100%;
-    height: 15px;
-    color: white;
+    width: 244px;
+    height: 28px;
+    box-sizing: border-box;
     border: none;
-    margin-bottom: 5px;
-    margin-top: 2px;
-}
-.vscode-light .iamInput {
+    padding-left: 8px;
+    padding-right: 8px;
+    padding-top: 6px;
+    padding-bottom: 6px;
+    font-size: 13px;
+    font-weight: 400;
+}
+body.vscode-light .iamInput {
     color: black;
 }
-.vscode-dark .iamInput {
-    color: white;
+body.vscode-dark .iamInput {
+    color: #cccccc;
 }
 .regionSelect {
     background-color: var(--vscode-input-background);
-    width: 100%;
-    color: white;
+    width: 244px;
     margin-bottom: 5px;
     margin-top: 2px;
-}
-.vscode-light .regionSelect {
+    padding-left: 8px;
+    padding-right: 8px;
+    padding-top: 6px;
+    padding-bottom: 6px;
+    font-size: 13px;
+    font-weight: 400;
+}
+body.vscode-light .regionSelect {
```

Comment:
Not a blocker/relevant to this PR. But our root element has a **class** `body`, but then there is the actual `body` element which this is targeting. If you have time later we should probably rename the class `body` to something else like `appRoot` so that there isn't confusion